libmamba/src/core/download_impl.hpp

```diff
@@ -34,9 +34,11 @@ namespace mamba
         using on_success_callback = std::function<bool(DownloadSuccess)>;
         using on_failure_callback = std::function<bool(DownloadError)>;
 
-        explicit DownloadAttempt(const DownloadRequest& request);
+        DownloadAttempt() = default;
```

Comment:
I'm not totally sure but I think to link properly you need `DownloadAttempt::DownloadAttempt()=default;`  in the cpp (moving the definition in the cpp).
If you get no link issues in msvc, ignore this comment.